docs/Cluster-VictoriaMetrics.md

```diff
@@ -470,8 +470,28 @@ This strategy allows upgrading the cluster without downtime if the following con
 - The updated config / upgraded binary is compatible with the remaining components in the cluster.
   See the [CHANGELOG](https://docs.victoriametrics.com/CHANGELOG.html) for compatibility notes between different releases.
 
-  If at least a single condition isn't met, then the rolling restart may result in cluster unavailability
-  during the config update / version upgrade. In this case the following strategy is recommended.
+If at least a single condition isn't met, then the rolling restart may result in cluster unavailability
+during the config update / version upgrade. In this case the following strategy is recommended.
+
+#### Improving re-routing performance during restart
```

Comment:
nit: I think we don't need extra section here - it should be listed as a bullet-point  in `Updating / reconfiguring cluster nodes`

Reply:
I'd also recommend to make it prompt: just state what vmstorage does and how user can action on this. Plus link to the issue.